README.md

```diff
@@ -24,6 +24,31 @@ videojs('player_id').ready(function () {
 
       certificateUrl: '/path/to/certificate',
       licenseUrl: '/path/to/license',
+
+      // Optional settings
```

Comment:
Ideally, options would be separated and described in an options section.

See https://github.com/chemoish/videojs-bif#options or whatever similar.